lib/streamlit/elements/multiselect.py

```diff
@@ -182,6 +191,8 @@ def multiselect(
             is still empty space for it above the widget (equivalent to label="").
             If "collapsed", both the label and the space are removed. Default is
             "visible". This argument can only be supplied by keyword.
+        max_selections : int
+            The max selections that can be selected at a time. This argument can only be supplied by keyword.
```

Comment:
Maybe move the `This argument can...` down one line to make it a bit shorter here.